### PR TITLE
Support `def scalaJSSourceMap: T[Boolean]`

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

**Review by lefou — COMMENTED** [review]
Do we have some test that checks for properly generated sourcemap file?

**Comment by lolgab** [comment]
> Do we have some test that checks for properly generated sourcemap file?

Added a test that checks the sourceMap to be actually disabled

File: scalajslib/src/mill/scalajslib/ScalaJSModule.scala [hunk 1]
```diff
@@ -232,6 +235,8 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
   def scalaJSOptimizer: Target[Boolean] = T { true }
 
+  def scalaJSSourceMap: Target[Boolean] = T { true }
```
Comment (lefou) [inline scalajslib/src/mill/scalajslib/ScalaJSModule.scala:238]: Can we add some docs here?

**Review by lefou — APPROVED** [review]
Looks good to me. Thank you, Lorenzo!

**Pull request merged**